isacc_messaging/api/isacc_record_creator.py

```diff
@@ -117,6 +117,12 @@ def convert_communicationrequest_to_communication(self, cr):
             result = self.send_twilio_sms(message=expanded_payload, to_phone=target_phone)
 
         except TwilioRestException as ex:
+            # The error is raised when the user has unscubscribed 
+            # mark this user as inactive for the future CRs
+            sms_telecom_entry = next((entry for entry in patient.telecom if entry.system.lower() == 'sms'))
+            sms_telecom_entry.period.end = FHIRDate(datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'))
```

Comment:
isn't sms_telecom_entry.period an array of Periods?

Reply:
I think it has max cardinality of 1 (so, 1 period?).
![image](https://github.com/uwcirg/isacc-messaging-service/assets/94884910/dc1dac03-621b-45ef-81a8-ad44f1db5b5d)

Reply:
i was wrong, period is a 0..1 cardinality